ethereum/api/src/test/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/QosTimerTest.java

```diff
@@ -14,67 +14,68 @@
  */
 package org.hyperledger.besu.ethereum.api.jsonrpc.internal;
 
-import static org.assertj.core.api.Assertions.assertThatThrownBy;
 import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
 
-import java.util.concurrent.TimeoutException;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
 
 import io.vertx.core.Vertx;
-import io.vertx.ext.unit.Async;
-import io.vertx.ext.unit.TestContext;
-import io.vertx.ext.unit.junit.VertxUnitRunner;
-import org.junit.Ignore;
-import org.junit.Test;
-import org.junit.runner.RunWith;
+import io.vertx.junit5.VertxExtension;
+import io.vertx.junit5.VertxTestContext;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
 
-@RunWith(VertxUnitRunner.class)
+@ExtendWith(VertxExtension.class)
 public class QosTimerTest {
   static Vertx vertx = Vertx.vertx();
+  private final VertxTestContext testContext = new VertxTestContext();
 
-  @Ignore("fails on CI with short timeouts and don't want to slow test suite down with longer ones")
+  @Disabled(
+      "fails on CI with short timeouts and don't want to slow test suite down with longer ones")
   @Test
-  public void shouldExecuteConsecutivelyAtTimeout(final TestContext ctx) {
+  public void shouldExecuteConsecutivelyAtTimeout() throws InterruptedException {
     final long TEST_QOS_TIMEOUT = 100L;
     final int INVOCATIONS_COUNT = 2;
-    final Async async = ctx.async(INVOCATIONS_COUNT);
+    final CountDownLatch latch = new CountDownLatch(INVOCATIONS_COUNT);
```

Comment:
@macfarla This change is made due to no support for VertxTestContext.async() in vertx junit5 lib - https://vertx.io/docs/vertx-junit5/java/
And since CountDownLatch behaves similarly to TestContext.async() I thought to replace the code with it